src/sentry/reprocessing2.py

```diff
@@ -514,6 +514,8 @@ def start_group_reprocessing(
         referrer="reprocessing2.start_group_reprocessing",
     )["data"][0]["times_seen"]
 
+    sentry_sdk.set_extra("event_count", event_count)
```

Comment:
Is this `set_extra` limited to the scope of the `@instrumented_task`? Same question for `set_tag` below.

Reply:
Yes